Scenario/Editor/PromptWindow/PromptWindow.cs

```diff
@@ -1,28 +1,25 @@
 using Newtonsoft.Json;
-using RestSharp;
 using System;
 using System.Collections;
 using System.Collections.Generic;
-using System.Threading.Tasks;
 using Unity.EditorCoroutines.Editor;
 using UnityEditor;
 using UnityEngine;
-using UnityEngine.Networking;
-using Method = RestSharp.Method;
 using System.Globalization;
 using System.Linq;
 
 public class PromptWindow : EditorWindow
 {
-    internal static List<ImageDataStorage.ImageData> generatedImagesData = new List<ImageDataStorage.ImageData>();
+    internal static List<ImageDataStorage.ImageData> generatedImagesData = new();
 
-    public PromptWindowUI promptWindowUI;
+    private PromptWindowUI promptWindowUI;
 
     private string inferenceId = "";
     private EditorCoroutine inferenceStatusCoroutine;
     private bool processReceivedUploadImage = false;    // for main thread receiving callback
     private byte[] pngBytesUploadImage = null;
-    
+    private string fielName;
```

Comment:
`fielName` => `fileName`

Reply:
Not sure it's misspelled? What's the reason or this?